apps/frontend/src/components/global/ExportASFFModal.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <v-dialog v-model="showingModal" width="500px">
+    <template #activator="{on}">
+      <LinkItem
+        key="export_csv"
+        text="Export as ASFF"
+        icon="mdi-file-document-multiple"
+        @click="showModal"
+        v-on="on"
+      />
+    </template>
+    <v-card>
+      <v-card-title class="headline"> Export as ASFF </v-card-title>
+      <v-card-text>
+        <v-text-field v-model="awsAccountId" label="AWS Account ID">
+          <v-tooltip slot="append" bottom>
+            <template #activator="{on}">
+              <v-icon dark v-on="on"> mdi-information-outline </v-icon>
+            </template>
+            <span
+              >To find your Account ID, log into the AWS management console<br />
+              In navigation bar, click the dropdown containing your username<br />
+              Your Account ID will be listed next to "My Account"
+            </span>
+          </v-tooltip>
+        </v-text-field>
+
+        <v-text-field v-model="target" label="Target Name">
+          <v-tooltip slot="append" bottom>
+            <template #activator="{on}">
+              <v-icon dark v-on="on"> mdi-information-outline </v-icon>
+            </template>
+            <span
+              >The target name is a user-defined string that allows tracking
```

Comment:
Does the Target Name becomes the result set Name in AWS? if so may want to stat it here.